events/wc_events/wc_lordaeron_events.txt

```diff
@@ -0,0 +1,410 @@
+﻿namespace = lordaeron 
+
+##############################################
+# 1001 - 1099: Reclamation of Lordaeron Events
+##############################################
+
+lordaeron.1001 = { 
+
+	type = character_event
+	title = lordaeron.1001.title
+	desc = lordaeron.1001.desc
+	theme = stewardship
+	override_background = market
+
+	immediate = { 
+		title:b_lordaeron.title_province = { 
+			remove_building = ruined_palace_01
+			add_special_building = ruined_palace_02 
+		}
+		title:c_lordaeron ={
+			set_county_culture = culture:forsaken
+		}
+	}
+
+	right_portrait = { 
+		character = root
+		animation = steward
+	}
+
+	trigger = { 
+		OR = { 
+			is_of_forsaken_like_culture = yes 					#make sure i have the title...
+			title:b_lordaeron.holder = { 						#or that I gave out the title to someome...
+				self_or_liege_is_of_forsaken_like_culture = yes 
+				any_liege_or_above = {this = root }
+			}
+		}
+	}
+
+	on_trigger_fail = { 
+		trigger_event = lordaeron.1002
+	}
+
+	option = { 
+		name = lordaeron.1001.opt.a
+
+		show_as_tooltip = { 
+			custom_tooltip = lordaeron.1001.tooltip.a
+		}
+
+		hidden_effect = { 
+			global_var:sylvanas_character = { 
+				trigger_event = { # replace with on-action for redirect?? or just use trigger_fail???
```

Comment:
on `immediate` check is she dead and if yes then set new character scope for global variable sylvanas?

Reply:
i can't get this to work at all and i'm running out of time here. i am going to pin this and get all your other recommendations done. i'll start work on the story cycle immediately and if it makes it in, it makes it in, otherwise people will just have to survive to see this happen >.<